### PR TITLE
[Workers] Request and response limit updates

### LINKED ISSUE #1361: Document request size limits for Workers
### Expected Behavior

https://developers.cloudflare.com/workers/platform/limits includes documentation about the maximum size of an HTTP request to Workers (e.g. 30MB, 500MB, unlimited).

### Actual Behavior

There is no documentation explaining what the maximum size for a request can be.

### Section that requires update

https://developers.cloudflare.com/workers/platform/limits#worker-limits

### Additional information

Google App Engine [documents](https://cloud.google.com/appengine/docs/standard/php/how-requests-are-handled#quotas_and_limits) the max request and response size.

### PR DESCRIPTION
**Pull request by deadlypants1973**
Closes https://github.com/cloudflare/cloudflare-docs/issues/1361

### CODE REVIEW

**Review by lukeed — APPROVED** [review]

File: products/workers/src/content/platform/limits.md [hunk 1]
```diff
@@ -22,9 +22,15 @@ pcx-content-type: concept
 
 </TableWrap>
 
-## Upload size limits
+## Request limits
 
-Cloudflare has network-wide limits on the body size of uploads (HTTP POST/PUT/PATCH requests) which depend on your Cloudflare plan (this is separate from your Workers plan). All uploads larger than your plan limit will be rejected with an HTTP 413 error code ("Request entity too large"). Cloudflare Enterprise customers can [contact Cloudflare support](https://support.cloudflare.com/hc/en-us/articles/200172476) to request a larger upload limit. Learn more about [Cloudflare plans](https://www.cloudflare.com/plans/).
+URLs have a limit of 16KB.
+
+Request headers observe a total limit of 32KB, but each heaer is limited to 16KB.
```
Comment (kodster28) [inline products/workers/src/content/platform/limits.md]:
```suggestion
Request headers observe a total limit of 32KB, but each header is limited to 16KB. 
```

File: products/workers/src/content/platform/limits.md [hunk 2]
```diff
@@ -37,6 +43,10 @@ Cloudflare has network-wide limits on the body size of uploads (HTTP POST/PUT/PA
 
 </TableWrap>
 
+## Response limits
+
+Cloudflare does not enforce response limts, but cache limits for [Cloudflare's CDN are observed](https://support.cloudflare.com/hc/en-us/articles/200172516-Understanding-Cloudflare-s-CDN#h_51422705-42d0-450d-8eb1-5321dcadb5bc). Maximum file size is 512MB for Free, Pro, and Business customers and 5GB for Enterprise customers.
```
Comment (kodster28) [inline products/workers/src/content/platform/limits.md]:
```suggestion
Cloudflare does not enforce response limits, but cache limits for [Cloudflare's CDN are observed](https://support.cloudflare.com/hc/articles/200172516#h_51422705-42d0-450d-8eb1-5321dcadb5bc). Maximum file size is 512MB for Free, Pro, and Business customers and 5GB for Enterprise customers.
```

File: products/workers/src/content/platform/limits.md [hunk 3]
```diff
@@ -22,9 +22,15 @@ pcx-content-type: concept
...
+## Request limits
 
-Cloudflare has network-wide limits on the body size of uploads (HTTP POST/PUT/PATCH requests) which depend on your Cloudflare plan (this is separate from your Workers plan). All uploads larger than your plan limit will be rejected with an HTTP 413 error code ("Request entity too large"). Cloudflare Enterprise customers can [contact Cloudflare support](https://support.cloudflare.com/hc/en-us/articles/200172476) to request a larger upload limit. Learn more about [Cloudflare plans](https://www.cloudflare.com/plans/).
+URLs have a limit of 16KB.
+
+Request headers observe a total limit of 32KB, but each heaer is limited to 16KB. 
+
+Cloudflare has network-wide limits on the request body size. This limit is tied to your Cloudflare Account's plan, which is separate from your Workers plan. When the request body size of your POST/PUT/PATCH requests exceed your plan's limit, the request is rejected with a `(403) Request entity too large` error.
+
+Cloudflare Enterprise customers may contact [Cloudflare Support](https://support.cloudflare.com/hc/en-us/articles/200172476) to request a limit beyond 500MB.
```
Comment (kodster28) [inline products/workers/src/content/platform/limits.md]:
```suggestion
Cloudflare Enterprise customers may contact [Cloudflare Support](https://support.cloudflare.com/hc/articles/200172476) to request a limit beyond 500MB.
```

Comment (kodster28) [reply products/workers/src/content/platform/limits.md]: Just FYI, better to remove the `en-us` part of the URL so folks can get to their native language if they have that configured on their profile + the article is actually translated.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**